js/twitterApi.js

```diff
@@ -60,13 +66,65 @@ $(document).ready(function(){
 
 			// So that no person is selected more than once
 			twitterUsersArr.splice(rand, 1);
-
-			console.log(twitterUsersArr.length);
 		}
 
```

Comment:
I feel for JS files we can leave whitespace lines in between. it makes the code much more readable. Too many lines together is not easy on the eyes for navigation.